model/src/main/scala/org/broadinstitute/dsde/rawls/model/ErrorReportable.scala

```diff
@@ -1,7 +1,6 @@
-package org.broadinstitute.dsde.rawls.dataaccess
+package org.broadinstitute.dsde.rawls.model
```

Comment:
I really feel like all the ErrorReport stuff goes in util but not ready to do that yet. That will come when PerRequest is moved to util.